DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Login
+@model List<ChooseACentreViewModel>
+@{
+  ViewData["Title"] = "Choose A Centre";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/login/chooseACentre.css")" asp-append-version="true">
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+
+    <h1 id="page-heading" class="nhsuk-heading-xl">Choose a Centre</h1>
+    <p class="nhsuk-body-l">
+      You are registered at more than one centre. Please choose the centre you would like to sign in to.
+    </p>
+
+    <table role="table" class="nhsuk-table-responsive">
+      <thead role="rowgroup" class="nhsuk-table__head">
+      <tr role="row">
+        <th role="columnheader" class="" scope="col">
```

Comment:
Are these empty classes required?

Reply:
The empty classes shouldn't be doing anything, but they are present in the examples on the NHS guidelines. https://service-manual.nhs.uk/design-system/components/table guessing we would prefer them to be removed?

Reply:
Yep - no need for empty classes, they'll just make everything else harder to read (not sure why the example has empty classes 🤔 ).